account_invoice_line_price_subtotal_gross/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account invoice line gross price subtotal
```

Comment:
Hi @codingforfun Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

document title: add a line before